.travis.yml

```diff
@@ -131,21 +131,22 @@ before_script:
   - sleep 5  # give starman a chance to start
   - utils/test/Is_LSMB_running.sh  # fail early if starman is not running
   - nginx -c /tmp/nginx.conf
-  - if [ "${DOJO_BUILT}" != "" ]; then sed -i -e "s#dojo_built *= *(\d)+#dojo_built={dojo_built}#" ledgersmb.conf ; fi
 
 script:
-  - RELEASE_TESTING=1
-    PGUSER=postgres PGPASSWORD=test
-    LSMB_TEST_DB=1 LSMB_NEW_DB=lsmb13installtest
-    LSMB_BASE_URL="http://127.0.0.1:5000"
-      prove -r --source Perl
-            --ext=.t --ext=.pg --ext=.feature
-            --source pgTAP   --pgtap-option dbname=lsmb13installtest
-                             --pgtap-option username=postgres
-                             --pgtap-option suffix=.pg
-            --source Feature --feature-option extensions=.feature
-                             --feature-option tags=~@wip
-            t/ xt/
+  - if [ "${DOJO_BUILT}" != "" ]; then sed -i -e "s#dojo_built *= *(\d)+#dojo_built={dojo_built}#" ledgersmb.conf ; fi
```

Comment:
This line was moved to the `before_script` section on purpose (it's preparing testing, not part of the test invocation itself). Did you move it back because you disagree?